src/pages/index.js

```diff
@@ -38,26 +41,79 @@ const modalImageCloseButton = previewImageModal.querySelector(
   "#modal__image-close-button"
 );
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "65e7222e-9077-40ab-ad47-86d360e310e2",
+    "Content-Type": "application/json",
+  },
+});
+
+const editAvatarPopup = new PopupWithForm({
+  popupSelector: "#edit-avatar-modal",
+  handleFormSubmit: handleAvatarFormSubmit,
+});
+editAvatarPopup.setEventListeners();
+
+const profileAvatarForm = document.forms["avatar-form"];
```

Comment:
Good job that you get the form from `document.forms`

Please, get other froms like that as well (`profileEditForm` and `addCardFormElement`)